applications/matter_bridge/src/bridge.zap

```diff
@@ -3207,10 +3207,10 @@
               "side": "server",
               "type": "bitmap32",
               "included": 1,
-              "storageOption": "RAM",
+              "storageOption": "External",
```

Comment:
Why did you change it from RAM to External? I'm not saying it's wrong, I just see FeatureMap is stored in RAM for all clusters, so I wonder why you decided to change it only for this one.

Reply:
Recently, we have done it in this way for DoorLock sample, and AFAIR it worked properly so I aligned all samples to that change. @doublemis1 What do you think?

Reply:
I am not aware about these solution, what suits best.
However, I agree with Arek that it should be aligned across all samples/clusters/endpoints